stack.yaml

```diff
@@ -2,13 +2,13 @@
 # For more information, see: http://docs.haskellstack.org/en/stable/yaml_configuration.html
 
 # Specifies the GHC version and set of packages available (e.g., lts-3.5, nightly-2015-09-21, ghc-7.10.2)
-resolver: lts-5.1
+resolver: lts-5.9
 
 # Local packages, usually specified by relative directory name
 packages:
 - '.'
 # Packages to be pulled from upstream that are not in the resolver (e.g., acme-missiles-0.3)
-extra-deps: []
+extra-deps: [aeson-0.11.0.0]
```

Comment:
Right, the cleanest solution would be not using `>=0.11` while it's not in the long term support package, a second possible way but still not too good would be using a nightly package set instead of lts, like [this](https://www.stackage.org/nightly-2016-03-28), which actually features `aeson-0.11.1.4`, but still needs to be updated with every new release.